expense/src/server/index.ts

```diff
@@ -26,42 +30,54 @@ async function run() {
   });
 
   app.post('/create', function(req, res) {
-    allExpenses[req.body.id] = expenseState.created;
+    allExpenses.set(req.body.id, expenseState.created);
 
     return res.json({ ok: 1 });
   });
 
   app.post('/action', function(req, res) {
-    const oldState = allExpenses[req.body.id];
+    const id = req.body.id;
```

Comment:
What do you think about this syntax?
```suggestion
    const { id } = req.body;
```

Reply:
Also, if you're not validating the request, at least add a comment that in "real code" this is required.